.github/workflows/swift-toolchain.yml

```diff
@@ -768,13 +760,23 @@ jobs:
           path: ${{ github.workspace }}/SourceCache/swift-driver
           show-progress: false
 
-      - name: Install Swift Toolchain
-        uses: compnerd/gha-setup-swift@b6c5fc1ed5b5439ada8e7661985acb09ad8c3ba2 # main
+      - name: Install Swift Toolchain (macOS)
+        if: matrix.os == 'Darwin'
+        uses: compnerd/gha-setup-swift@6c9f2db7c3155c57fe35f160bcd5cf5859b9c1ba # main
         with:
           host_arch: ${{ inputs.build_arch }}
```

Comment:
Unrelated to your changes but this is not a valid input for `gha-setup-swift`. Could you remove it at the same time?

Reply:
Done